graphql/schema.graphql

```diff
@@ -81,6 +81,8 @@ type Query {
 type Mutation {
   addFavoriteProject(identifier: String!): Project!
   removeFavoriteProject(identifier: String!): Project!
+  attachProjectToRepo(id: String!): Project!
+  detachProjectFromRepo(id: String!): Project!
```

Comment:
attachUserToProject? replace id: with `projectId` ?

Reply:
Are you suggesting a rename of attachProjectToRepo to attachUserToProject? That actually isn't what's happening in this function; we're turning on the option of defaulting to repo for the branch project.

Reply:
oops didn't actually read what `DetachFromRepo` and `AttachFromRepo` did. I assumed they did something with the user that was passed into them, to give the user some permission.